server_extensions_extra/src/main/resources/anker.svg

```diff
@@ -0,0 +1,8 @@
+
+		<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="3057.3248407643314" height="3382.1656050955417">
```

Comment:
anker.png - change the resolution of png file. see other currency png files to find correct resolution.
anker.svg - linearGradient will not most likely supported. please test it on ATM.